frontend/src/Styles/Player.module.css

```diff
@@ -4,9 +4,10 @@
     border-radius: 50%;
     background-color: rgb(43, 43, 43);
     color: white;
-    display: flex;
+    display: inline-flex;
```

Comment:
@ateagit please let me know if this change breaks your player component. I did this to allow for multiple player components that adjust themselves to fit the screen depending on the number present.

Reply:
Yep, should be fine. I'll make a seperate pr for table if it breaks.